app/views/home/index.html.haml

```diff
@@ -230,6 +230,13 @@
         %a{href:logo[:url], target: "_blanc"}
           %img{src: asset_path(logo[:img_src])}
 
+-# This section is hidden, used to initialize the cache of federation portals status
```

Comment:
always replace comments with a better function names (a good video about it https://www.youtube.com/watch?v=wXaC0YvDgIo&list=PL9wALaIpe0Py6E_oHCgTrD6FvFETwJLlx)

Reply:
Thank you